FEXCore/Source/Interface/IR/Passes/RegisterAllocationPass.cpp

```diff
@@ -611,8 +611,8 @@ bool ConstrainedRAPass::Run(IREmitter* IREmit_) {
         if (Class->Allocated & (1u << Reg.Reg)) {
           OrderedNode* Old = Class->RegToSSA[Reg.Reg];
 
-          LOGMAN_THROW_AA_FMT(IsOld(Old), "RegToSSA invariant");
-          LOGMAN_THROW_AA_FMT(IsOld(Node), "Haven't remapped this instruction");
+          LOGMAN_THROW_A_FMT(IsOld(Old), "RegToSSA invariant");
```

Comment:
remind me the difference?

Reply:
AA_FMT will do a `__builtin_assume` in release builds. A_FMT is just an "assert format".
Since `IsOld` has a side-effect it throws a warning.

Reply:
AA = `Assume + Assert`
A = `Assert`